pyproject.toml

```diff
@@ -46,6 +46,9 @@ codecarbon = "^2.5.0"
 optional = true
 [tool.poetry.group.extra-mlips.dependencies]
 alignn = "^2024.5.27"
+sevennet = "^0.9.1"
```

Comment:
Whether or not we manage to use extras nicely for the current default MLIPs (such that `pip install janus-core` installs MACE, M3GNet and CHGNet by default, but `pip install janus-core[mace]` would only install MACE), perhaps we should use extras for these optional MLIPs, rather than bundling them all together.

I haven't tested this, but in this case I think we might want something like:

```
alignn = { version = "^2024.5.27", optional = true }
sevennet = { version = "^0.9.1", optional = true }


[tool.poetry.extras]
alignn= ["alignn"]
sevennet = ["sevennet "]
```

We'd need to update the tutorial to note this too.

See: https://python-poetry.org/docs/pyproject/#extras (note the behaviour differs slightly to the current implementation via groups)